### PR TITLE
CLI: add sessions key

### PR DESCRIPTION
**Pull request by sbesson**
See https://trello.com/c/dyRbaNiM/19-key-omero-sessions-key

Implement a simple RFE to return the current session key via the CLI. To test this PR:
- check the new integration test is green
- from the command line check the output of `bin/omero sessions key` is null if logged out and the current session key if logged in
- test a simple workflow reusing the session key:

  ```
   bin/omero login USER@HOSTNAME
   bin/omero -C -s HOSTNAME -k $(bin/omero sessions key) COMMAND
```

### CODE REVIEW

File: components/tools/OmeroPy/src/omero/plugins/sessions.py [hunk 1]
```diff
@@ -820,11 +823,19 @@ def run(self):
...
         store = self.store(args)
         srv, usr, uuid, port = store.get_current()
         if srv and usr and uuid:
             self.ctx.out(str(store.dir / srv / usr / uuid))
 
+    def key(self, args):
+        """Return the key associated to the current active sessions"""
+        store = self.store(args)
+        srv, usr, uuid, port = store.get_current()
+        if uuid:
```
Comment (joshmoore) [inline components/tools/OmeroPy/src/omero/plugins/sessions.py]: `else: self.ctx.die(321, "No key found")` ?

Comment (sbesson) [inline components/tools/OmeroPy/src/omero/plugins/sessions.py]: Can do. In that case, I was mirroring the behavior of `session file` which does not throw a CLI exception is no local session file is found. Either of them is fine.

Comment (sbesson) [inline components/tools/OmeroPy/src/omero/plugins/sessions.py]: Can do. In that case, I was mirroring the behavior of `session file` which does not throw a CLI exception is no local session file is found. Either of them is fine.

File: components/tools/OmeroPy/src/omero/plugins/sessions.py [hunk 2]
```diff
@@ -820,11 +823,19 @@ def run(self):
             t.event.set()
 
     def file(self, args):
+        """Return the file associated to the current active sessions"""
```
Comment (mtbc) [inline components/tools/OmeroPy/src/omero/plugins/sessions.py]: "associated /with/" is more idiomatic (but "related to")

**Comment by mtbc** [comment]
https://ci.openmicroscopy.org/job/OMERO-5.1-merge-integration-python/lastCompletedBuild/testReport/test.integration.clitest.test_sessions/TestSessions/ looks good.

**Comment by mtbc** [comment]
Functionally seems to work excellently, it's a good addition to the CLI that I've been wanting.

**Comment by sbesson** [comment]
@mtbc: pushed

**Comment by mtbc** [comment]
:+1: Looks good to me!

**Comment by sbesson** [comment]
@jburel: re https://github.com/openmicroscopy/openmicroscopy/pull/4005#discussion_r36017509, I suggest we get this merged as it is and rebase onto develop. I will open a `develop`-only PR changing the return code of `sessions file/key` to return an error if empty and we can decide whether we want to backport it into `dev_5_1`.

**Comment by jburel** [comment]
merging. @joshmoore's comment will be addressed in another PR

**Comment by sbesson** [comment]
--rebased-to #4022

**Pull request merged**